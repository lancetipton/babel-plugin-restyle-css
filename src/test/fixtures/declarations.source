
reStyle`transform: ${getTransform()}; border-radius: ${20}`;