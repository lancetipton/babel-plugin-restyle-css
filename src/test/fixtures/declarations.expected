
reStyle({
  transform: getTransform(),
  borderRadius: 20
});